packages/browser-repl/src/components/editor.tsx

```diff
@@ -22,7 +23,7 @@ interface EditorProps {
   onClearCommand(): void | Promise<void>;
   onSigInt(): Promise<boolean>;
   operationInProgress: boolean;
-  setInputRef?(ref: { editor?: HTMLElement }): void;
+  setInputRef?(ref: { container?: HTMLElement }): void;
```

Comment:
I'm not entirely sure what's going on here. I basically just made it compile. Looking at the types the only html element is container, not editor. But so far I haven't found evidence in history somewhere to see where that changed and if it did.

Reply:
It’s totally possible that the type here should have been `Editor` from ace – that’s also something with a `.focus()` method

Reply:
Yup. Figured that out eventually. :)